espresso/core/java/androidx/test/espresso/device/dagger/DeviceControllerModule.kt

```diff
@@ -39,19 +40,24 @@ internal class DeviceControllerModule {
   @Provides
   @Singleton
   fun provideActionContext(): ActionContext {
-    // TODO(b/203570026) Initialize ActionContext depending on whether the test is an
-    // instrumentation
-    // test or Robolectric
     return InstrumentationTestActionContext()
   }
 
   @Provides
   @Singleton
   fun provideDeviceController(): DeviceController {
-    if (isTestDeviceAnEmulator()) {
-      return EmulatorController(getEmulatorControllerStub())
+    val platformDeviceController: androidx.test.platform.device.DeviceController? =
```

Comment:
@brettchabot what about using `import androidx.test.platform.device.DeviceController as PlatformDeviceController` to simplify this code block? Kotlin supports `import full_class_name as` syntax for using same classes at different package in one target class.

Reply:
Oh, the `DeviceController` is migrated to platform one at https://github.com/android/android-test/pull/1260.